lunchtime10/LTP_PyTorch.ipynb

```diff
@@ -0,0 +1,914 @@
kCLDiJHfVyfGwwVqZv+MJ78a9NFcTZmJixdJsi/pjzRNW1LnW+ccal88zARMwARMwARMwARMwARM4egIWoI+elY80ARMwgSMSwM/oB6+4ShZ4dXXswfDqyy9F684jnugDTMAETMAETMAETMAETMAETMAETMAETOB9SMAC9PuwUJ0lEzCBd49AJflwrS+/pViGv/rSiwF/vWw65mACJmACJmACJmACJmACJmACJmACJmACpyIBC9CnYqk7zyZgAu8Ygbe1eeCf7/+drJ43auOvnXHTxXfsYo7YBEzABN4HBNh4E/dF/B1vSOJJxXX88R1veny+CZiACZiACZiACZiACZiA9skyBBMwARMwgcIRwF/pyuVLCxehYzIBEzCB9zmBvdpPoIr+O7B/nzYQPnBcucXP80H9xyaaBw7sP664fLIJmIAJmIAJmIAJmIAJmEBhCFiALgxHx2ICJmACJmACJmACJlBGAgcOHAjPP/NkqFSpUrSAXrF8WRljyD581AvPhspVqsS4Vq9alf2jP5mACZiACZiACZiACZiACbwrBCxAvyvYfVETMAETMAETMAETMIGDBw+GhfPnFQzEooULChaXIzIBEzABEzABEzABEzABEygMgfKFicaxmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmEA2AQvQ2Tz8yQRMwARMwARMwARMwARMwARMwARMwARMwARMwARMoEAELEAXCKSjMQETMAETMAETMAETMAETMAETMAETMAETMAETMAETyCZgATqbhz+ZgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAkUiIAF6AKBdDQmYAImYAImYAImYAImYAImYAImYAImYAImYAImYALZBCxAZ/PwJxMwARMwARMwARMwARMwARMwARMwARMwARMwARMwgQIRsABdIJCOxgRMwARMwARMwARMwARMwARMwARMwARMwARMwARMIJuABehsHv5kAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZQIAIWoAsE0tGYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAlkE7AAnc3Dn0zABEzABEzABEzABEzABEzABEzABEzABEzABEzABApEwAJ0gUA6GhMwARMwARMwARMwARMwARMwARMwARMwARMwARMwgWwCFqCzefiTCZiACZiACZiACZiACZiACZiACZiACZiACZiACZhAgQhYgC4QSEdjAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiaQTcACdDYPfzIBEzABEzABEzABEzABEzABEzABEzABEzABEzABEygQAQvQBQLpaEzABEzABEzABEzABEzABEzABEzABEzABEzABEzABLIJWIDO5uFPJmACJmACJmACJmACJmACJmACJmACJmACJmACJmACBSJgAbpAIB2NCZiACZiACZiACZiACZiACZiACZiACZiACZiACZhANgEL0Nk8/MkETMAETMAETMAETMAETMAETMAETMAETMAETMAETKBABCxAFwikozEBEzABEzABEzABEzABEzABEzABEzABEzABEzABE8gmYAE6m4c/mYAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJFIiABegCgXQ0JmACJmACJmACJmACJmACJmACJmACJmACJmACJmAC2QQsQGfz8CcTMAETMAETMAETMAETMAETMAETMAETMAETMAETMIECEbAAXSCQjsYETMAETMAETMAETMAETMAETMAETMAETMAETMAETCCbgAXobB7+ZAImYAImYAImYAImYAImYAImYAImYAImYAImYAImUCACFqALBNLRmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJZBOwAJ3Nw59MwARMwARMwARMwARMwARMwARMwARMwARMwARMwAQKRMACdIFAOhoTMAETMAETMAETMAETMAETMAETMAETMAETMAETMIFsAhags3n4kwmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYQIEIWIAuEEhHYwImYAImYAImYAImYAImYAImYAImYAImYAImYAImkE3AAnQ2D38yARMwARMwARMwARMwARMwARMwARMwARMwARMwARMoEAEL0AUC6WhMwARMwARMwARMwARMwARMwARMwARMwARMwARMwASyCViAzubhTyZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAgUiYAG6QCAdjQmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYQDYBC9DZPPzJBEzABEzABEzABEzABEzABEzABEzABEzABEzABEygQAQsQBcIpKMxARMwARMwARMwARMwARMwARMwARMwARMwARMwARPIJmABOpuHP5mACZiACZiACZiACZiACZiACZiACZiACZiACZiACRSIgAXoAoF0NCZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAiZgAtkEKmZ/9KdjIVC9eo1QpVq1cPDgwWM53eeYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmYgAmcYALlypULu3ftCju2bzvBVz61LmcBugDl3aBx49C0ecuwf9++AsTmKEzABEzABEzABEzABEzABEzABEzABEzABEzABN5pAhUqVgxrV68Ki+bPfacvdUrHbwG6AMW/Z/eesH3btnBg//4CxOYoTMAETMAETMAETMAETMAETMAETMAETMAETMAE3mkC5StUCHt2736nL3PKx28BugBVYPXK5YE/BxMwARMwARMwARMwARMwARMwARMwARMwARMwARMwgRIC3oSwhIXfmYAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJmIAJFJCABegCwnRUJmACJmACJmACJmACJmACJmACJmACJmACJmACJmACJQQsQJew8DsTMAETMAETMAETMAETMAETMAETMAETMAETMAETMIECErA
```

Comment:
For people without GPUS (and the ci runners on github) you could add this somewhere early in the notebook:
```
device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
```
then replace any `.cuda()` with `.to(device)` so that the notebook also works without a gpu.

Reply:
Hi Liam,

thanks for the great suggestions! I changed the notebook according to your suggestions. If there is something else that makes it more compatible with the LTP repo let me know!

Reply:
Thanks @kierandidi !
One last thing, could you run `pip-compile` to update requirements.txt from requirements.in (if you don't have this command you can install it with `pip install pip-tools`). Then (hopefully) the ci should work.
Thanks again for your very nice talk!

Reply:
Ok @lkeegan, seems to have worked now!

Reply:
great, thanks again!